packages/manager/modules/pci/src/projects/project/private-registry/api-url/index.js

```diff
@@ -0,0 +1,35 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import apiUrlComponent from './api-url.component';
+
+const moduleName = 'ovhManagerPciProjectPrivateRegistryApiUrl';
```

Comment:
Should be "lazyloaded"

Reply:
It is a very small component. Just a modal with one component. Lazyload will be overkill for this

Reply:
The modal has only one field to copy the URL, better to be not lazy loaded, what do you think?

Reply:
I see your point, but let's say we have several small components like this (like for a cloud instance); this means those several small components will be loaded even if they are not used. Imo, even if it is small it shouldn't be loaded if it is not needed